Cisco/cisco-ise/tests/test_network_info_events1.json

```diff
@@ -0,0 +1,21 @@
+{
+  "input": {
+    "message": "0000738292 7 0 2025-01-09 09:28:49.914 +01:00 0043796555 5200 NOTICE Passed-Authentication: Authentication succeeded, ConfigVersionId=186, Device IP Address=2.3.4.5, DestinationIPAddress=1.2.3.4, DestinationPort=1812, UserName=testUserName, Protocol=Radius, NetworkDeviceName=MGCUH1-3, User-Name=testUserName, NAS-IP-Address=2.3.4.5, NAS-Port=1001, Service-Type=Framed, Framed-MTU=1449, State=37CPMSessionID=XXXXXXXXXXXXXX\\;36SessionID=MGDFSISE2/515448264/543038\\;, Called-Station-ID=a0:ec:f9:50:e2:90:DFS, Calling-Station-ID=bc:83:85:d8:88:c6, NAS-Identifier=MGCUH1-3, NAS-Port-Type=Wireless - IEEE 802.11, NAS-Port-Id=Capwap2, EAP-Key-Name=, cisco-av-pair=service-type=Framed, cisco-av-pair=audit-session-id=XXXXXXXXXXXXXX, cisco-av-pair=method=dot1x, cisco-av-pair=cisco-wlan-ssid=DFS, Airespace-Wlan-Id=5, NetworkDeviceProfileName=Cisco, NetworkDeviceProfileId=aaaaaaaaaaaaaaaaaaaaa,#015"
```

Comment:
From the content, we can extract additional information from the event.
I think we can extract the username, the destination IP address, the destination port, the device IP address (`source.ip`?), the protocol (`network.protocol`), the Called Station ID (`destination.mac` with the `mac_address` filter) and the Calling Station ID (`source.mac` with the `mac_address` filter)